plugins/MantisCoreFormatting/MantisCoreFormatting.php

```diff
@@ -158,6 +160,21 @@ function formatted( $p_event, $p_string, $p_multiline = true ) {
 
 		$t_string = $p_string;
 
+		if( null === $s_markdown ) {
+			$s_markdown = plugin_config_get( 'process_markdown' );
+		}
+
+		# Process Markdown and return content, no further text processing.
+		if( ON == $s_markdown ) {
+			if( $p_multiline ) {
+				$t_string = MantisMarkdown::convert_text( $t_string );
```

Comment:
As there is no further text processing, you can return the value without assigning to a variable.